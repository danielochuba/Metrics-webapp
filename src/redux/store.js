import { configureStore } from '@reduxjs/toolkit';
import countrySlice from './country/countrySlice';

const store = configureStore({
  reducer: {
    movies: countrySlice,
  },
});

export default store;
