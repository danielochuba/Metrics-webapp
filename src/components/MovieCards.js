import React from 'react';

import {
  Container, Row,
} from 'react-bootstrap';
import { Provider } from 'react-redux';
import MovieCard from './MovieCard';
import '../css/navbar.css';
import store from '../redux/store';

function MovieCards() {
  return (
    <Container style={{ marginTop: '1rem' }}>
      <Row>
        <Provider store={store}>
          <MovieCard />
        </Provider>
      </Row>
    </Container>
  );
}

export default MovieCards;
