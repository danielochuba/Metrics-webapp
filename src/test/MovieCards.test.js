import React from 'react';
import { render } from '@testing-library/react';
import { Provider } from 'react-redux';
import MovieCards from '../components/MovieCards';
import store from '../redux/store';

describe('MovieCards component', () => {
  test('renders without errors', () => {
    render(<Provider store={store}><MovieCards /></Provider>);
  });
});

test('matches the snapshot', () => {
  const { container } = render(<MovieCards />);
  expect(container).toMatchSnapshot();
});
