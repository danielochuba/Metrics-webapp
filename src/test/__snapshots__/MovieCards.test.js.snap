// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`matches the snapshot 1`] = `
<div>
  <div
    class="container"
    style="margin-top: 1rem;"
  >
    <div
      class="row"
    />
  </div>
</div>
`;
